packages/odyssey-design-tokens/src/font.json

```diff
@@ -1,5 +1,28 @@
 {
   "font": {
+    "color": {
+      "body": {
+        "comment": "Base text color used for most typography",
+        "value": "{hue.neutral.900.value}"
+      },
+      "heading": {
+        "comment": "Base text color used for headings",
+        "value": "{hue.neutral.900.value}"
+      },
+      "bodyInverse": {
+        "comment": "Inverse text color typically used on a colorful background",
+        "value": "{hue.neutral.white.value}"
+      },
+      "primary": {
+        "comment": "Branded text color, primarily used within button",
+        "value": "{palette.primary.main.value}"
+      },
+      "sub": {
```

Comment:
+1

We use `subtitle` and `caption` in other places. It'd be nice to get consistent there.

MUI uses `subtitle`, but we can override that / add a new type.

Reply:
To look at it as a system... is the recessive gray text 600 and we also need a 700? Is the recessive use (600?) secondarily used vs 900 as primary? 

When 700 is added so we have a scale of 3....
Main (900)
Light (700)
Lightest (600_
OR
Primary (900)
Secondary (now 700)
Tertiary (600; this use is unique in our taxonomy so far)

Prominent could be interpreted as more important than primary/main...like headline/subtitle?

The other use of primary and secondary color is for icon tokens. If we change text, do icon need to shift so we describe values and use in a similar way?

Reply:
I'm not too attached to having second-order variants share similar names unless it's straightforward. I *would* like to stay away from hierarchies when it comes to functional tokens because they're arbitrary.

`body`, `heading`, and `bodyInverse` seem clear and likely won't show up anywhere outside of `typography`. 

`danger` makes to me, since the use case is clear.

I think `primary` was named as such simply because it matches our `palette.primary`, but it doesn't really spell out the case. `action` might be more intuitive. 

Re: `sub`, I think I'm most partial to either `subtitle` (potentially confusing but matches MUI) and `caption` (not a 1:1 use case, but doesn't rely on hierarchy).

This contrasts with Button, where the `primary` variant indicates an in-UI hierarchy of importance.

Reply:
@edburyenegren-okta I don't know if I can get typography scale/use (like subtitle, body small) aligned with color tokens... Should I try to align color uses with color? Suggestions above either align with icon color tokens (primary, secondary) or hue/primary, error, warning, success (main, light, lightest)

A third idea: 
900 could easily be body
but 
700 = tag labels, table column headers, and areas of system design ...label feels closer than subheading or header to describe use of these cases
600 = secondary, supporting content in the UI to set it back a level hierarchically ....body light (related to body small?)

Reply:
> I'm not too attached to having second-order variants share similar names unless it's straightforward. I _would_ like to stay away from hierarchies when it comes to functional tokens because they're arbitrary.
> 
> `body`, `heading`, and `bodyInverse` seem clear and likely won't show up anywhere outside of `typography`.
> 
> `danger` makes to me, since the use case is clear.
> 
> I think `primary` was named as such simply because it matches our `palette.primary`, but it doesn't really spell out the case. `action` might be more intuitive.
> 
> Re: `sub`, I think I'm most partial to either `subtitle` (potentially confusing but matches MUI) and `caption` (not a 1:1 use case, but doesn't rely on hierarchy).
> 
> This contrasts with Button, where the `primary` variant indicates an in-UI hierarchy of importance.

I think I can get behind use of color but can we tweak a few: 
`body`
`bodyInverse`
`error` (more consistent vs danger)
`link` (if that's what we mean by action for typography color)
- if we are naming by use, then I can't tell where subtitle or caption are used because we really don't have them? Would 'label' work (for 700 use case of tag label and table column header?

Reply:
`error` works for me! Button might be the only case we use "danger" because it's a user action rather than user-facing info.

`link` might be misleading because we also use it for Tabs and some Buttons.

`caption/sub/secondary` - I just did a quick inventory. We use this for: 

- Field hints
- legalese
- actual captions (e.g. the `figcaption` under an image or graph)
- keyboard keys / `kbd`
- Chips (disabled text & icons)
- IconButton (disabled)
- Link (monochrome hover)
- List Subheaders (e.g. option groups in Select or Menu)
- Table footers, headers, and sort indicators

This isn't representative of the latest Rebrand UI color usage; however, the fact that they may change is probably a good indicator that they may need different tokens even if they coincidentally share the same value.